### DISCUSSION TITLE
Expo SDK 48 Support

### DISCUSSION THREAD

**Discussion by jerielng** [reactwg/react-native-new-architecture#127] [Documentation]
Hi, is this note about Expo incompatibility in the [documentation](https://reactnative.dev/docs/next/the-new-architecture/use-app-template#development-environment) still relevant? Based on the Expo 48 [release notes](https://blog.expo.dev/expo-sdk-48-ccb8302e231), it appears that the New Architecture is now compatible with some minor exceptions:
```
Support for the New Architecture/Fabric in all Expo packages except expo-dev-client. Expo Go also does not support the New Architecture. We’re not quite ready to recommend that you use this for your app yet since you’re likely to run into a number of issues (eg: third-party modules that don’t yet support the new architecture).
```
So I just wanted to confirm if this was still the case. Thanks!

**Comment by cortinico** [comment]
You're right that the note is not relevant anymore. @Simek @Kudo can I ask you to confirm and potentially update the note in the docs with a reference on your end?

**Reply by jerielng** [reply]
Thanks for the clarification, @cortinico!
